gui/test/integration/cypress/support/page_files/NodeSearchInput.ts

```diff
@@ -0,0 +1,102 @@
+class NodeSearchSuggestionLink {
+  constructor(
+    private readonly index: number,
+    private readonly searchSuggestions: NodeSearchSuggestions
+  ) {}
+
+  get() {
+    return this.searchSuggestions.get().find("ul>a").eq(this.index);
+  }
+
+  click() {
+    this.get().click();
+
+    return this;
+  }
+}
+
+class NodeSearchAllResultsLink {
+  constructor(private readonly searchSuggestions: NodeSearchSuggestions) {}
+
+  get() {
+    return this.searchSuggestions.get().find("ul>a:last-child");
+  }
+
+  click() {
+    this.get().click();
+
+    return this;
+  }
+}
+
+export class NodeSearchSuggestions {
+  // The component `div` is actually created in the root component
+  // so it has 'no parent elements'
+  readonly selector: string = "[data-cy=nodeSearchSuggestions]";
+
+  get() {
+    return cy.get(this.selector);
+  }
+
+  assertNoResults() {
+    this.get().find("ul").children().should("have.length", 1);
```

Comment:
Prefer data-cy, this code is likely fragile.

Reply:
Removed since new setup seems to show no box when there is no results